charts/simpledataexchanger/values.yaml

```diff
@@ -26,7 +26,7 @@ sdepostgresql:
   enabled: true
   fullnameOverride: "product-sde-postgres"
   auth:
-    postgresPassword: "default"
+    postgresPassword: ""
```

Comment:
Is there a function that generates a random password when this is left empty?

Reply:
yes, It will generate random password.